### ISSUE TITLE
Update and share journal content tracking spreadsheet

### ISSUE THREAD

**Issue by carakey** [osulp/ojs#31]

**Comment by carakey** [comment]
A full listing of OJS2 contents has been output here: https://docs.google.com/spreadsheets/d/1cV8taNhO5kgVeyJRV0GWeL-ta5khHgSIsg_0pLTG0oM/edit#gid=1311041921 (the "2023 All OJS2 Content" worksheet).

Same data also available as a CSV: 
[2023_All_OJS2_Content.csv](https://github.com/osulp/ojs/files/11127917/2023_All_OJS2_Content.csv)

The data shows 103 issues and 749 submissions that we should expect to have migrated, for a total of 852 items. 

In column Q I've done a lookup against the OJS3 submission IDs in the "IDlookup" worksheet, which gave 516 hits. I used these to construct what _should_ be the correct URL in OJS3 for the item, if it has been migrated. Spot checking these, it's looking pretty good for #16. 

Of the 336 items not found in the IDlookup sheet, 150 are from 2020-2023, so probably can be attributed to publication after our initial export work. 

This worksheet can be used to track the remaining migration work and content QA.

**Comment by carakey** [comment]
I reviewed my notes and the journals3 live content, and have updated the spreadsheet further. The numbers have been updated, and there were 68 items not included in the IDlookup that have indeed been migrated; 268 items remain. 

The migration stands as such:  

- 32 issues containing 236 submissions (268 total content items) remain to be migrated. Of these: 
  - 2 issues / 18 submissions failed to import to OJS3, possibly because of their size 
  - 2 single submissions failed to export from OJS2 because of embedded multimedia files
  - 5 issues / 20 submissions had been exported but not migrated before upgrading from 3.1 to 3.3 
  - 8 issues / 69 submissions were exported and migrated but deleted trying to troubleshoot the "white screen of death (WSOD)" issue in 3.1.x (see #10 )
  - 18 issues / 126 submissions were created and published after our initial export work
- The rest of the content has been migrated, so there are 584 items -- 71 issues and 513 submissions -- that are ready to be QA'd.
